packages/perseus/src/util.ts

```diff
@@ -1,10 +1,10 @@
+import {type Range} from "@khanacademy/perseus-core";
 import {KhanAnswerTypes} from "@khanacademy/perseus-score";
 import _ from "underscore";
 
 import * as GraphieUtil from "./util.graphie";
 
 import type {PerseusStrings} from "./strings";
-import type {Range} from "@khanacademy/perseus-core";
```

Comment:
I'm surprised this moved.. could we move it back to the `type` imports section here?

Reply:
I think it's because the `type` keyword got moved inside the brackets. I think this was because I had imported something else from core at one point that was not a type, so I moved the type keyword inside the brackets and the linter moved it as a result. Moving it out allowed it to stay back where it was before.